app/models/person.rb

```diff
@@ -273,4 +273,20 @@ def self.custom_types
       .order(:custom_type)
       .pluck('distinct custom_type')
   end
+
+  def self.create_with_omniauth(auth)
+    create! do |person|
+      person.provider = auth['provider']
+      person.uid = auth['uid']
+      person.first_name = auth['info']['first_name']
+      person.last_name = auth['info']['last_name']
+      person.email = auth['info']['email']
+      person.can_sign_in = true
```

Comment:
I think this needs to be conditional based on the "Signup Approval Email" setting. [See here](https://github.com/churchio/onebody/blob/master/config/settings.yml#L66-L68) and [here](https://github.com/churchio/onebody/blob/master/app/models/signup.rb#L124-L126).

In fact, I think this method should be integrated into the `Signup` model just so it's all in one place. Thoughts?